README.md

```diff
@@ -240,19 +269,25 @@ Example that follows both rules:
 
 ### Variables
 
-Use variables when it makes sense.
+Use variables when it makes sense. Variables exist for most of the wallets and some other product names. It is preferred to use the variable instead of, e.g., browser wallet.
```

Comment:
```suggestion
Use variables when it makes sense.
Variables exist for most of the wallets and some other product names.
It is preferred to use the variable instead of, e.g., browser wallet.
```
If we want to have that one sentence per line rule.
Could be applied several places in this readme.

Reply:
In this case it needs to be said explicitly because I still get a lot of browser wallet even though the variable has been there for months now.